### PR TITLE
XW-3636 | autoplay on mobile

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3636
https://github.com/Wikia/mobile-wiki/pull/113
https://github.com/Wikia/app/pull/13286

@Wikia/x-wing

### CODE REVIEW

File: sample.html [hunk 1]
```diff
@@ -74,10 +74,22 @@
             window.pp.mb.publish(OO.EVENTS.WIKIA.SHOW_AD_FULLSCREEN_TOGGLE, false);
           }
         }, false, this);
+
+        // that's a hack for autoplay on mobile for VPAID ads
```
Comment (rybmat) [inline sample.html:78]: can we mark it as "For the future" or at least mention that vpaids are not fully supported now?

**Review by rybmat — CHANGES_REQUESTED** [review]

**Review by rybmat — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

File: js/views/wikiaAdScreen.js [hunk 2]
```diff
@@ -16,16 +16,17 @@ var WikiaAdScreen = React.createClass({
...
+      || this.props.controller.state.controlBarVisible
+      || this.props.playerState == CONSTANTS.STATE.PAUSE;
 
     var playbackControlItemTemplates = {
       "controlBar": <ControlBar {...this.props}
                                 controlBarVisible={showControlBar}
                                 playerState={this.props.playerState}
                                 isWikiaAdScreen={true}
                                 showAdTimeLeft={this.props.controller.state.showAdTimeLeft}
-                                showAdFullScreenToggle={this.props.controller.state.showAdFullScreenToggleToggle}
```
Comment (hakubo) [inline js/views/wikiaAdScreen.js:28]: :)

File: sample.html [hunk 3]
```diff
@@ -40,24 +40,24 @@
...
   var vastUrl =
     'https://pubads.g.doubleclick.net/gampad/ads?output=xml_vast3&env=vp&gdfp_req=1&unviewed_position_start=1&iu=%2F5441%2Fwka.life%2F_project43%2F%2Farticle%2Fpremium%2FFEATURED_VIDEO&sz=640x480&url=http%3A%2F%2Fproject43.wikia.com%2Fwiki%2FSyntheticTests%2FPremium%2FFeaturedVideo%3FAbTest.FEATURED_VIDEO_AUTOPLAY%3DCONTROL&correlator=1927378681583883&cust_params=wsi%3Doxap%26s0%3Dlife%26s0v%3Dlifestyle%26s0c%3Dtech%26s1%3D_project43%26s2%3Darticle%26ab%3D49_162%2C52_170%2C78_336%26ar%3D4%3A3%26artid%3D355%26dmn%3Dwikiacom%26hostpre%3Dproject43%26skin%3Doasis%26lang%3Den%26wpage%3Dsynthetictests%2Fpremium%2Ffeaturedvideo%26ref%3Ddirect%26esrb%3Dteen%26geo%3DPL%26pv%3D6%26u%3Dryo8cyllq%26ksgmnt%3D%26top%3D1k%26ah%3D9000%26pos%3DFEATURED_VIDEO%26src%3Dpremium&eid=41351088&sdkv=h.3.173.0&sdki=3c0d&scor=264486118542786&adk=3826960879&osd=2&frm=0&sdr=1&mpt=Ooyala&mpv=1.0&afvsz=200x200%2C250x250%2C300x250%2C336x280%2C450x50%2C468x60%2C480x70&ged=ve4_td164_tt158_pd164_la0_er419.141.574.441_vi0.0.941.1000_vp100_eb24171';
+  var autoplay = !OO.isIphone && Cookies.get('html5-skin.autoplay') !== '0';
```
Comment (hakubo) [inline sample.html]: OO.isIphone - should that be here?

File: sample.html (hunk 1 above)
Comment (hakubo) [reply sample.html:78]: and by not perfectly? what does that mean?
can we have a small list what is not working?
will we remember in 3 months what that means?

Comment (rogatty) [reply sample.html:78]: We have a spreadsheet with issues. I would leave it as it is - it'll be a bit less work if we decide to fix VPAID.

**Review by hakubo — APPROVED** [review]

**Pull request merged**